src/Gui/CommandDoc.cpp

```diff
@@ -1574,7 +1574,7 @@ void StdCmdPlacement::activated(int iMsg)
 
 bool StdCmdPlacement::isActive(void)
 {
-    return (Gui::Control().activeDialog()==0);
+    return (Selection().getCompleteSelection().size() > 0) || (Gui::Control().activeDialog() != 0);

```

Comment:
So the command is active if (selection > 0 OR active dialog open)... shouldn't it be (selection > 0 AND no active dialog)?

Reply:
Indeed I was too fast on this and this is inappropriate. Fortunately you saw it.
Should be better now. I applied same rule as Std_TransformManip.
There could be still improved filtering on this, but it already brings some improvement and consistency.

Reply:
@yorikvanhavre Actually last time I messed up my git commands and it was not fixed. Now is OK (for sure). :)